docs/deploying/unix.md

```diff
@@ -53,12 +53,39 @@ really, you probably want some process monitoring using tools like
 [monit](http://mmonit.com/monit/),
 [god](http://godrb.com/),
 [bluepill](https://github.com/arya/bluepill),
-[upstart](http://upstart.ubuntu.com/).
+[upstart](http://upstart.ubuntu.com/),
+[systemd](http://freedesktop.org/wiki/Software/systemd/).
 
 For starting at boot, you can create an init script for appropriate your UNIX
 distribution, or if you are using one the process monitoring tools above, make
 sure it boots at startup.
 
+### Example systemd unit file
+
+Place in e.g. `/etc/systemd/system/hubot.service`, then `systemctl daemon-reload` and `service hubot start`.
+
+```ini
+[Unit]
+Description=Hubot
+Requires=network.target
+After=network.target
+
+[Service]
+Type=simple  
+WorkingDirectory=/opt/hubot
+User=hubot
+
+Restart=always
+RestartSec=10
+
+Environment=HUBOT_aaa=xxx HUBOT_bbb=yyy
+
+ExecStart=/opt/hubot/bin/hubot --adapter zzz
+
+[Install]
+WantedBy=multi-user.target
+```
```

Comment:
There currently aren't example scripts for the other tools. Does it make sense to include on only for systemd?